src/Plugin/Comments.php

```diff
@@ -4,19 +4,22 @@
 
 use AdCaptcha\Widget\AdCaptcha\AdCaptcha;
 use AdCaptcha\Widget\Verify\Verify;
+use AdCaptcha\Plugin\Plugin;
 use WP_Error;
 
-class Comments {
+class Comments extends Plugin {
```

Comment:
When I read this I feel it would be nice if Plugin was called `AdCaptchaPlugin`. Makes it clearer to me what is what

```suggestion
class Comments extends AdCaptchaPlugin {
```